<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatchedFilterWizard</class>
 <widget class="QWidget" name="MatchedFilterWizard">
  <layout class="QVBoxLayout" name="verticalLayout_root">
   <property name="leftMargin"><number>0</number></property>
   <property name="topMargin"><number>5</number></property>
   <property name="rightMargin"><number>0</number></property>
   <property name="bottomMargin"><number>0</number></property>
   <item>
    <widget class="QWidget" name="widgetAddButton" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_add">
      <property name="spacing"><number>10</number></property>
      <item>
       <widget class="QPushButton" name="addTargetButton">
        <property name="text"><string>Add Color</string></property>
        <property name="font"><font><pointsize>11</pointsize></font></property>
        <property name="minimumSize"><size><width>120</width><height>32</height></size></property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation"><enum>Qt::Horizontal</enum></property>
        <property name="sizeHint" stdset="0"><size><width>40</width><height>20</height></size></property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="targetsContainer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="targetsLayout">
      <property name="spacing"><number>6</number></property>
      <property name="leftMargin"><number>0</number></property>
      <property name="topMargin"><number>0</number></property>
      <property name="rightMargin"><number>0</number></property>
      <property name="bottomMargin"><number>0</number></property>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_bottom">
     <property name="orientation"><enum>Qt::Vertical</enum></property>
     <property name="sizeType"><enum>QSizePolicy::Expanding</enum></property>
     <property name="sizeHint" stdset="0"><size><width>20</width><height>40</height></size></property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
